src/executables/util_service.c

```diff
@@ -1836,6 +1855,38 @@ process_server (int command_type, int argc, char **argv, bool show_usage, bool c
   return status;
 }
 
+/*
+ * process_server_start_single_node - start both a page and a transaction server,
+ *                                     the page server must be started before transaction server
+ *
+ * return:
+ */
+static int
+process_server_start_single_node (char *server_name, int command_type)
+{
+  int status, pid;
```

Comment:
Separate each variable on its own line and initialize.
```suggestion
  int status = NO_ERROR;
  int pid = 0;
```